src/test/java/com/marginallyclever/makelangelo/TranslatorTests.java

```diff
@@ -49,4 +59,46 @@ public void changeLanguageTest() {
 			Translator.saveConfig();
 		}
 	}
+
+	@Test
+	public void searchSimpleStringMissingKeyUsedAsArgumentForTranslatorGetInTheSrcCode() {	    
+	    try {
+		//Pre requis
+		Translator.start();
+		
+		String baseDirToSearch = "src" + File.separator + "main" + File.separator + "java";
+		System.out.printf("PDW=%s\n", new File(".").getAbsolutePath());
+		File srcDir = new File(".", baseDirToSearch);
+		try {
+		    System.out.printf("srcDir=%s\n", srcDir.getCanonicalPath());
+		} catch (IOException ex) {
+		    ex.printStackTrace();
```

Comment:
If you don't know what to do with the exception, let it flow. You are in a test, thus, the runner will fails with a beautiful error message

Reply:
Thanks !